File: solutions/javascript/reverse-string/1/reverse-string.js
//
// This is only a SKELETON file for the 'Reverse String' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const reverseString = (str) => {
  var ret = '';
  for(var i=str.length;i>=0;i--) ret+=str.charAt(i);
  return ret;
};

File: solutions/java/reverse-string/1/src/main/java/ReverseString.java
class ReverseString {

    String reverse(String inputString) {
        String s = "";
        int l = inputString.length();
        for (int j = 0; j<l; j++ ) {
                s = s + inputString.substring((l - j) - 1, l - j);
        }
        return (s);
    }
  
}